client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -114,9 +115,21 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 							onClick={ () => {
 								dispatch( submitSignupStep( { stepName: 'confirm' }, { siteConfirmed: siteId } ) );
 								if ( siteUpgrading.required ) {
```

Comment:
I think we can have only one dispatch line before this with `upgrade_required: siteUpgrading.required`, there is any reason that prevents that?

Reply:
Addressed at https://github.com/Automattic/wp-calypso/pull/60513/commits/b75555565fde17e591715c64ec63d77a3c465ec9